.idea/encodings.xml

```diff
@@ -32,8 +32,13 @@
     <file url="file://$PROJECT_DIR$/websocket/jetty10/src/filter/resources" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/websocket/jetty10/src/main/java" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/websocket/jetty10/src/main/resources" charset="UTF-8" />
+    <file url="file://$PROJECT_DIR$/websocket/jetty12/src/filter/resources" charset="UTF-8" />
+    <file url="file://$PROJECT_DIR$/websocket/jetty12/src/main/java" charset="UTF-8" />
+    <file url="file://$PROJECT_DIR$/websocket/jetty12/src/main/resources" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/websocket/spi/src/filter/resources" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/websocket/spi/src/main/java" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/websocket/spi/src/main/resources" charset="UTF-8" />
+    <file url="file://$PROJECT_DIR$/../winstone/src/main/java" charset="UTF-8" />
+    <file url="file://$PROJECT_DIR$/../winstone/src/main/resources" charset="UTF-8" />
```

Comment:
This doesn't seem essential to this PR, but rather an unrelated cleanup. I would suggest removing this hunk from this PR and submitting it as a separate change against the default branch, where it can be merged promptly rather than stagnating on this project branch that can't be merged for several weeks or months.